landoapi/models/landing.py

```diff
@@ -0,0 +1,115 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from landoapi.models.storage import db
+from landoapi.phabricator_client import PhabricatorClient
+from landoapi.transplant_client import TransplantClient
+
+TRANSPLANT_JOB_STARTED = 'started'
```

Comment:
Could you add a comment explaining that these strings are known values copied from the Transplant service API?